test/unit/test_proxies.py

```diff
@@ -11,3 +11,16 @@ def test_list_proxies():
     proxy_list = proxlist.list_proxies()
 
     assert proxy_list == proxlist.proxies.PROXY_LIST
+
+    
+def test_proxy_dict():
+    proxy_dict = proxlist.proxy_dict()
+    http = proxy_dict.get('http', None)
```

Comment:
```suggestion
    http = proxy_dict.get('http')
```
None is the implicit default here, no need to include

Reply:
Same with the one below it.